### DISCUSSION TITLE
[Feature request]: TOPAS-imaging extension?

### DISCUSSION THREAD

**Discussion by wizrads** [OpenTOPAS/OpenTOPAS#87] [Feature-Request] [answered]
### Feature request

Hello, 

I was wondering if anyone is able to locate the TOPAS-imaging package mentioned here?
[https://pubmed.ncbi.nlm.nih.gov/36930985/](https://pubmed.ncbi.nlm.nih.gov/36930985/)

I am unable to find it :(

Thank you so much in advance!

Best,
Joey

### TOPAS category

Physics

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by leehoy** [comment, marked as answer]
Hi, I'm sorry for the delay in the release of the extension.
It is now available at https://github.com/OpenTOPAS/TOPAS-imaging.
